### DISCUSSION TITLE
Just wanted to say thank you!

### DISCUSSION THREAD

**Discussion by 1031982** [black7375/Firefox-UI-Fix#826] [General]
The overhaul that Mozilla has done is fine for touch screens, but it's such a waste when one is using a keyboard and mouse. This customization, at least in my opinion, saves a lot of space and generally makes for a more pleasurable view.

Thank you for your continued work on this.

**Comment by black7375** [comment]
Thank you.
I'm a fan of Firefox too and want to keep it around for as long as possible. :smile: 

I'm definitely going to try switching to a web extension and auto-updating next year.

**Reply by rubcap** [reply]
@black7375 Same here. Thank you SO much for your dedicated work on improving the Firefox UI 🧡 .
I don't know if I could use Firefox without some of your customizations.

I just read https://github.com/black7375/Firefox-UI-Fix/wiki/%5BArticle%5D-0.-Firefox-UI-UX-history ... trip down memory lane!
